magprime/templates/emails/confirm_shirt_size.html

```diff
@@ -0,0 +1,7 @@
+<p>Hello {{ attendee.first_name }},</p>
+
+<p>Our records show that you will be receiving a t-shirt at MAGFest 2016. Please take a moment to review <a href="http://magfest.org/images/2016swag/shirtsizeguide.png">this sizing chart</a>, and then check that you have the right shirt size selected on your <a href="{{ c.URL_BASE }}/preregistration/confirm?id={{ attendee.id }}">registration information page</a> in the the "Shirt Size" dropdown. If you'd like to change your size, please do so using the dropdown, and then click the "Update My Info" button. If you're satisfied with the shirt size that is already selected, then no further action is necessary.</p>
+
+<p>Please make sure that we have your desired shirt size no later than <b>January 15th, 2016</b>, so that we can ensure we have enough stock for your size. Your shirt will be available for pickup at the MAGFest merchandise booth.</p>
+
+<p>Ryan Nichols<br>MAGFest Merch Department<br>merch@magfest.org</p>
```

Comment:
If it's alright with you, I'd prefer to hide that gargantuan confirmation url with an html href